src/main/java/com/example/demo/services/AccountService.java

```diff
@@ -25,11 +31,17 @@
 	private UsersRepository usersRepo;
 	private IHashUtil hashUtil;
 	private IJwtUtil jwtUtil;
+	private JavaMailSender mailSender;
 	
-	public AccountService(IHashUtil hashUtil,IJwtUtil jwtUtil, UsersRepository userRepo) {
+	private String subject="Reset your Auction Purple password";
+	private String content="Follow link below to set new password for your Auction purple account";
+	private String link="http://localhost:8080/newPassword?token=";
```

Comment:
don't use localhost:8000, will be very dificult when deployed to heroku to manage this